modules/ROOT/partials/integrations/react-bundling.adoc

```diff
@@ -33,22 +32,17 @@ import 'tinymce/plugins/media';
 import 'tinymce/plugins/nonbreaking';
 import 'tinymce/plugins/table';
 import 'tinymce/plugins/help';
+import 'tinymce/plugins/help/js/i18n/keynav/en';
 
 // Content styles, including inline UI like fake cursors
-/* eslint import/no-webpack-loader-syntax: off */
-import contentCss from '!!raw-loader!tinymce/skins/content/default/content.min.css';
-import contentUiCss from '!!raw-loader!tinymce/skins/ui/oxide/content.min.css';
```

Comment:
How is the CSS managed in Tinymce 7 that you can avoid this step? I assume this has all been tested but I'm naturally a bit worried since I can't understand how it would import these CSS files correctly?

Reply:
Those lines are replaced with side effect imports that run:
```js
tinymce.Resource.add('content/default/content.css', "body{font-family:-apple-system,BlinkMacSystemFont ...")
tinymce.Resource.add('ui/default/content.css', ".mce-content-body .mce-item-anchor ...")
```